roles/kubernetes/master/templates/kubeadm-config.v1alpha3.yaml.j2

```diff
@@ -7,11 +7,15 @@ nodeRegistration:
 {% if kube_override_hostname|default('') %}
   name: {{ kube_override_hostname }}
 {% endif %}
+{% if inventory_hostname in groups['kube-master'] and inventory_hostname not in groups['kube-node'] %}
```

Comment:
Why is that necessary? Ain't the kubernetes/master role executed only on servers in kube-master group?